build.php

```diff
@@ -26,32 +26,45 @@
 use Seld\PharUtils\Timestamps;
 use Symfony\Component\Finder\Finder;
 
-require_once __DIR__.'/vendor/autoload.php';
+$dir = __DIR__;
```

Comment:
Could you put this back? I don't like the idea of making a variable for what's a constant anyway, and I doubt the build folder will ever change